Step_Functions/Epoching_Resting/Bad_Epochs.m

```diff
@@ -91,6 +91,7 @@
                 Clean_Epochs_Mask(bad_Kurtosis) = 0;
                 
                 % Probability open pop_jointprob
+                % I don't see the frequency range set to 1-30 Hz - perhaps this is defined elsewhere but just want to check.
```

Comment:
this is not the frequency range. the frequency is analysed in line 85, there it is set, so please remove line